ecommerce/extensions/iap/api/v1/utils.py

```diff
@@ -188,15 +191,16 @@ def apply_price_of_inapp_purchase(price, in_app_purchase_id, headers):
     if response.status_code != 200:
         raise AppStoreRequestException("Couldn't fetch price points")
 
-    nearest_low_price = nearest_low_price_id = 0
+    # Apple doesn't allow in app price > 1000
+    nearest_high_price = nearest_high_price_id = 1001
     for price_point in response.json()['data']:
         customer_price = float(price_point['attributes']['customerPrice'])
-        if nearest_low_price < customer_price <= price:
-            nearest_low_price = customer_price
-            nearest_low_price_id = price_point['id']
+        if nearest_high_price > customer_price >= price:
```

Comment:
question: what is the scenario where `price` is greater than `customer_price`?

Reply:
customer price range is 0-1000 and we are already filtering > 1000 cases. Therefore there is no chance for this scenario. 
Even if it occurs this function will return exception and we will not apply price for that product and will investigate issue afterwards.